config.yml

```diff
@@ -0,0 +1,67 @@
+# General
+serverName: UUID
+colors: true
+showLogo: true
+logLevel: INFO
+
+# Connectivity
+webServerEnabled: true
+tcpServerEnabled: true
+port: 6020
+host: 0.0.0.0
+tcpPort: 6021
+tcpHost: 0.0.0.0
+httpServer: null
```

Comment:
Does this need to be in the yaml file? The argument would have to be a http object which can't be used via a config file.